enhancements/cluster-logging/forward_to_cloudwatch.md

```diff
@@ -0,0 +1,281 @@
+---
+title: forward_to_cloudwatch
+authors:
+  - "@alanconway"
+reviewers:
+  - "@jcantrill"
+  - "@jeremyeder"
+approvers:
+creation-date: 2020-12-17
+last-updated: 2020-12-17
+status: implementable
+see-also:
+superseded-by:
+---
+
+# Forward to CloudWatch
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+[Amazon CloudWatch][aws-cw] is a hosted monitoring and log storage service.
+This proposal extends the `ClusterLogForwarder` API with an output type for CloudWatch.
+
+## Motivation
+
+Amazon CloudWatch is a popular log store.
+We have requests from external and  Red Hat-internal customers to support it.
+
+### Goals
+
+Enable log forwarding to CloudWatch.
+
+### Non-Goals
+
+Enable CloudWatch metric collection.
+
+## Proposal
+
+### CloudWatch streams and groups
+
+[CloudWatch][concepts] defines *log groups* and *log streams*. To paraphrase the documentation:
+
+>  A log stream is a sequence of log events that share the same source ... For example, an Apache access log on a specific host.
+
+>  Log groups define groups of log streams that share the same retention, monitoring, and access control settings ... For example, if you have a separate log stream for the Apache access logs from each host, you could group those log streams into a single log group called MyWebsite.com/Apache/access_log.
+
+In other words a *log stream* corresponds to the smallest distinct source of logs.
+A *log group* is a collection of related *log streams*.
+
+Cluster logging collects logs from two distinct types of log source:
+
+- *container logs* are logs from containers, collected via the container run-time.
+- *node logs* are logs from a cluster node's journald or syslogd, they do *not* come from any container.
+
+**Note**: Log *sources* are different from log *categories*.
+Category *application* includes only container logs.
+Categories *infrastructure* and *audit* include a mix of selected container and node logs.
+
+#### Generating log streams
+
+This proposal generates a unique *log stream* for each distinct container or node log source.
+The *log stream name* is constructed from meta-data including the container/node UUID.
```

Comment:
Naming for audit logs?  im guessing someting with the node id in it and the specific audit log?

Reply:
Updated to follow your prototype: fluent-key for containers, node-id  + fluent-key for node logs (infra, audit)